### DISCUSSION TITLE
Chapter-7: repo initialization error

### DISCUSSION THREAD

**Discussion by lordshashank** [Cyfrin/foundry-full-course-cu#26] [Q&A] [answered]
I cloned the repo of chapter-7 and ran ```forge build``` 
getting this error
```
Error: 
File cannot be resolved due to mismatch of file name case: "/home/lordforever/blockchain/foundry/foundry-f23/foundry-fund-me-f23/lib/forge-std/src/Console.sol": No such file or directory (os error 2).
 Found existing file: "/home/lordforever/blockchain/foundry/foundry-f23/foundry-fund-me-f23/lib/forge-std/src/console.sol"
 Please check the case of the import..
    --> "/home/lordforever/blockchain/foundry/foundry-f23/foundry-fund-me-f23/lib/foundry-devops/src/DevOpsTools.sol"
        "forge-std/Console.sol"
```
can someone help plz.
I am running it on windows wsl.

**Comment by alymurtazamemon** [comment]
@lordshashank I tested it out but it is working. Can you please try again?

<img width="663" alt="Screenshot 2023-06-09 at 4 03 39 PM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/56123405/276858fb-18b8-4c0e-87ae-24b63e52ab4f">

**Reply by lordshashank** [reply]
tried but not working in my wsl system.

**Reply by alymurtazamemon** [reply]
@lordshashank Did you try to create a new project with `forge init project-name`? See is this working?

**Reply by lordshashank** [reply]
What do you mean I cloned and just followed the readme

**Reply by alymurtazamemon** [reply]
@lordshashank I mean just create a new project, there is one dependency `foundry-devops` which is causing an issue in your system. Just create a new project and replace contracts, scripts and test files and download dependencies yourself to see about errors particular to your system.

**Comment by PatrickAlphaC** [comment]
Ah, I had the repo in the readme as `PatrickAlphaC` instead of `Cyfrin`. I just updated it. 

Could you try the steps with `github.com/Cyfrin/foundry-fund-me`?

**Reply by lordshashank** [reply]
still same error

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/100513286/fa7f53d5-381d-41d8-b8bc-bc09f53c896c)

btw I cloned the correct repo before as well.

**Reply by alymurtazamemon** [reply, marked as answer]
@lordshashank I found it, in the `foundry-devops/src/DevOpsTools.sol` there is import `import {console} from "forge-std/Console.sol";` where `Console.sol` file name has type, it should be `console.sol`

**Reply by lordshashank** [reply]
thanks, this worked.
